cves/CVE-2013-0844.yml

```diff
@@ -127,7 +125,12 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    An enourmous amont of time took place from the introduction of the vulnerability
+    to its fix. Over 100 commits were made to the file containing the vulnerability.
+    Enormous code refactoring took place with a large expansion of functionality being built in
```

Comment:
Could you provide some simplified code refactoring examples?

Reply:
I feel that adding information about the code refactoring would bloat the description of the lifetime and not provide benefit and understanding for the end consumer.